### ISSUE TITLE
ESP8266 SDK version for building?

### ISSUE THREAD

**Issue by iDoka** [godFather89/HAP.js#3]
Andrei, hello again!
Which one version of ESP8266 SDK did you use for building image?
I had tryed 0.9.5 but unsuccesful.
Unfortunately, In Makefile defined path to SDK dir without any version-related hints.

Thanks!
Dmitry

**Comment by andrei-tatar** [comment]
I think I used 1.0 or 1.1. Anyways, I think I slightly changed some header file. Could you please post the error(s) you are getting?

**Comment by iDoka** [comment]
I suppose that I found the root of my issue:
Since version  SDK v1.1.0_15_05_26 appears new functions for global enable/disable IRQ:

```
/* disable all interrupts */
#define ETS_INTR_LOCK() ets_intr_lock()

/* enable all interrupts */
#define ETS_INTR_UNLOCK() ets_intr_unlock()
```

Therefore the minimal version of SDK is v1.1.0

**Comment by andrei-tatar** [comment]
Yes. Check out the new repo: https://github.com/andrei-tatar/HAP.esp/tree/master/hap_ir
It's working with the latest SDK (1.4.1)
